config/sources/families/include/riscv64_common.inc

```diff
@@ -0,0 +1,37 @@
+#
+# SPDX-License-Identifier: GPL-2.0
+#
+# Copyright (c) 2013-2023 Igor Pecovnik, igor@armbian.com
+# Copyright (c) 2013-2023 Jacob Hrbek, kreyren@armbian.com
+#
+# This file is a part of the Armbian Build Framework
+# https://github.com/armbian/build/
+#
+
+# This file includes variables that are applied by defalt for the riscv64 architecture
+
+ARCH="riscv64"
+
+OPENSBI_COMPILE="yes"
+
+INITRD_ARCH="riscv"
+
+GOVERNOR="ondemand"
```

Comment:
@igorpecovnik How do you want governors? Seems that they are set to `performance` in other families instead of the seemingly more sane `ondemand`, i guess you want the boards to be as responsive as possible without limitations when first deployed with armbian? :thinking: